.env

```diff
@@ -0,0 +1,7 @@
+REACT_APP_API_KEY = AIzaSyBO1rPnoT8CTuvwV_X8W1givAozC06RX3c
```

Comment:
never ever ever commit your .env file!!!!!! Always use a .gitignore file and include it there